XRTK-Core/Packages/com.xrtk.core/Editor/MixedRealityToolkitInspector.cs

```diff
@@ -76,7 +76,8 @@ public override void OnInspectorGUI()
                 {
                     case 0:
                         EditorGUIUtility.PingObject(target);
-                        EditorUtility.DisplayDialog("Attention!", "No root profile for the Mixed Reality Toolkit was found.\n\nYou'll need to create a new one.", "OK");
+                        EditorApplication.delayCall += () =>
```

Comment:
I am not sure with these .delayCall registration. Same thing as with the package installer we talked about yesterday. Pretty sure this is causing memory leaks in the editor.

Reply:
They're just closures. Should be okay.

It's a lot harder to create memory leaks I'm c# unless you're playing with low level memory

Reply:
@FejZa can you find me an example usage of this delay call that doesn't use an anonymous delegate?

Reply:
Like in XRTK? E.g. here https://github.com/XRTK/WindowsMixedReality/blob/development/XRTK.WindowsMixedReality/Packages/com.xrtk.wmr/Editor/WindowsMixedRealityPackageInstaller.cs

Reply:
I don't know. To me this looks like every GUI frame we are registering an event handler. But I might be wrong here. I just live by the rule, if I subscribe, I unsubscribe.

Reply:
No just in general.

Reply:
Let's take this discussion to discord :)